### ISSUE TITLE
After add nodejs-mobile plugin the app not builds

### ISSUE THREAD

**Issue by simaoferreira24694** [janeasystems/nodejs-mobile#67]
* **Version**: node v4.2.6
* **Platform**: Ubuntu 16.04.1 x64
* **Subsystem**: Cordova 8.0.0

I'm creating an mobile app for Cordova Android 6.4.0:
`cordova platform add android@6.4.0`

Without the nodejs-mobile the app builds successfully.

I add the plugin like this:
`cordova plugin add nodejs-mobile-cordova`

I have the NDK installed and exported.

After add the nodejs-mobile I build the app with this:
`cordova -d build android`

With the "-d" is possible to get the follow information:
**No scripts found for hook "before_build".
No scripts found for hook "before_prepare".
Checking config.xml and package.json for saved platforms that haven't been added to the project
Config.xml and package.json platforms are the same. No pkg.json modification.
Package.json and config.xml platforms are different. Updating config.xml with most current list of platforms.
PlatformApi successfully found for platform android
Checking config.xml for saved plugins that haven't been added to the project
Checking for any plugins added to the project that have not been installed in android platform
No differences found between plugins added to project and installed in android platform. Continuing...
Generating platform-specific config.xml from defaults for android at /home/spaf94/Work/5DevelopersPlus1/tellspec-mobile-app/platforms/android/res/xml/config.xml
Merging project's config.xml into platform-specific android config.xml
Merging and updating files from [www, platforms/android/platform_www] to platforms/android/assets/www
Wrote out android application name "Demo" to /home/spaf94/Work/5DevelopersPlus1/tellspec-mobile-app/platforms/android/res/values/strings.xml
android-versionCode not found in config.xml. Generating a code based on version in config.xml (1.0.0): 10000
Wrote out Android package name "com.example.hello" to /home/spaf94/Work/5DevelopersPlus1/tellspec-mobile-app/platforms/android/src/com/example/hello/MainActivity.java
This app does not have launcher icons defined
This app does not have splash screens defined
This app does not have additional resource files defined
Prepared android project successfully
Executing script found in plugin nodejs-mobile-cordova for hook "after_prepare": plugins/nodejs-mobile-cordova/install/hooks/android/after-prepare-build-node-assets-lists.js
Resolving module name for q => q
Executing script found in plugin nodejs-mobile-cordova for hook "after_prepare": plugins/nodejs-mobile-cordova/install/hooks/both/after-prepare-patch-npm-packages.js**

In the last line is possible to see that the build stops when executing a nodejs-mobile script.
Can anyone help me?

**Comment by jaimecbernardo** [comment]
Hi @spaf94 ,

This could be related to cordova 8.x not being supported yet, as stated in the plugins requirements: https://github.com/janeasystems/nodejs-mobile-cordova#requirements

Have you tried it with cordova 7.x ?

**Comment by simaoferreira24694** [comment]
Yes, I tried with cordova 7.x and the result is the same. The strange is that this only happens in a virtual machine that I use, in my physical pc the problem not happens.

**Comment by jaimecbernardo** [comment]
Are you using the same version of node in the physical PC and VM?
You specified `node v4.2.6`, which was released in January 2016, though I'm not sure why this would be an issue.
This sounds like a development machine configuration issue.
Let us know if you're able to fix it.
